src/inc/conpty-universal.h

```diff
@@ -142,14 +289,42 @@ HRESULT CreateConPty(const std::wstring& cmdline,
 
     LPCWSTR lpCurrentDirectory = startingDirectory.has_value() ? startingDirectory.value().c_str() : nullptr;
 
+    std::vector<wchar_t> newEnvVars;
+    auto zeroNewEnv = wil::scope_exit([&] {

```

Comment:
nit: just learned that wil has `wil::SecureZeroMemory_scope_exit(void*, size_t)`!

Reply:
I'm aware of that method and considered using it.  However, as far as I can tell I can't use it here because wil doesn't capture the size by reference, and since the final size isn't known at the point I declare this, I can't use that method.  That's why I used a scope_exit instead.  Is my understanding correct?

Reply:
I believe it is. Thanks!